packages/import-notation/lib/helpers.js

```diff
@@ -0,0 +1,187 @@
+/**
+ * Helpers to test import-notation first part
+ */
+const tests = {
```

Comment:
Not sure about this object. Prob it's enough to make 2 simple functions

Reply:
I did it a group of functions like `templates` object after

Reply:
nvm, not a blocker.